src/structure/pure/Include.js

```diff
@@ -15,32 +15,45 @@
  * You should have received a copy of the GNU General Public License along
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
- *
- * @module Include 
- * @title  CometVisu Include 
  */
 
 
 /**
+ * Includes other configuration snippets. Note that this is no widget shown in your visualization it just gives
+ * you the opportunity to split you configuration into multiple files. If you have a large configuration file
+ * splitting it up might help to keep track of your configuration.
+ *
+ * @module structure/pure/Include
+ * @requires structure/pure/
  * @author Christian Mayer
- * @since 2012
+ * @since 0.8.0 (2012)
  */
 define( ['_common'], function( design ) {
   "use strict";
   var basicdesign = design.basicdesign;
   
   design.basicdesign.addCreator('include', {
-  maturity: design.Maturity.development,
-  create: function( page, path, flavour, type ) {
-    var p = $.get( $(page).attr('src') );
-    var p = $.ajax({
-      url: $(page).attr('src'),
-      dataType: 'xml',
-      async: false
-    });
-    var child = (p.responseXML.childNodes[0]);
-    return templateEngine.create_pages( child, path , flavour ); 
-  }
-});
+    maturity: design.Maturity.development,
+    /**
+     * Creates the widget HTML code
+     *
+     * @method create
+     * @param {Element} element - DOM-Element
+     * @param {String} path - internal path of the widget
+     * @param {String} flavour - Flavour of the widget
+     * @param {String} type - Page type (2d, 3d, ...)
+     * @return {String} HTML code
+     */
+    create: function( page, path, flavour, type ) {
+      var p = $.get( $(page).attr('src') );
+      var p = $.ajax({
```

Comment:
Yes this is what I mentioned on gitter about a month ago (and forgot about it then), will add a hint